Source/utils/display.cpp

```diff
@@ -243,11 +243,11 @@ bool SpawnWindow(const char *lpWindowName)
 		flags |= SDL_WINDOW_FULLSCREEN;
 	}
 
-	if (*sgOptions.Gameplay.grabInput) {
-		flags |= SDL_WINDOW_INPUT_GRABBED;
-	}
-
 	ghMainWnd = SDL_CreateWindow(lpWindowName, SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, windowSize.width, windowSize.height, flags);
+
+	if (ghMainWnd != nullptr)

```

Comment:
Please add a comment pointing to the SDL issue: https://github.com/libsdl-org/SDL/issues/962

Reply:
Added comment.